### PR TITLE
yoav - Rollers joystick commands (waiting for the funnel) (doi not merge)

### PR DESCRIPTION
**Pull request by bossyshrimp68**
dont merge until funnel and roller ocmmand merged!!!

### CODE REVIEW

**Review by tnrz135 — APPROVED** [review]

**Review by yonnbro — CHANGES_REQUESTED** [review]
Add a third and forth debug controllers and assign a joystick to each one

File: src/main/java/edu/greenblitz/robotName/OI.java [hunk 1]
```diff
@@ -33,11 +37,19 @@ public class OI {
...
+    private final SmartJoystick fourthJoystick;
+
     private OI() {
         mainJoystick = new SmartJoystick(RobotConstants.Joystick.MAIN);
         secondJoystick = new SmartJoystick(RobotConstants.Joystick.SECOND);
-        initButtons();
-        initializeDefaultCommands();
+        thirdJoystick = new SmartJoystick(RobotConstants.Joystick.THIRD);
+        fourthJoystick = new SmartJoystick(RobotConstants.Joystick.FOURTH);
+        //initButtons();
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/robotName/OI.java:49]: Why comment initbuttons and defaultcommands?

Comment (yonnbro) [reply src/main/java/edu/greenblitz/robotName/OI.java]: I meant leave the “official” controls and just add the new joysticks

**Review by yonnbro — APPROVED** [review]

**Comment by yonnbro** [comment]
Test master!

**Pull request merged**